versions.yaml

```diff
@@ -1,4 +1,6 @@
 supported:
+  - channel: '0.29'
```

Comment:
When I look at the previous release, seems like we keep one latest release - https://github.com/radius-project/radius/pull/6920/files

Reply:
Removed the older release and moved to deprecated section. Will update the Example link in release documentation page to reflect this.

Reply:
yeah we leave the option open in the future to support multiple releases but for now we have just been keeping one.

Reply:
one official release that is - we should keep the previous release in this case since it is a release candidate. See https://github.com/radius-project/radius/pull/6920/files for an example